public/extensions/demo/states/services/details/demo/software/software.html

```diff
@@ -0,0 +1,23 @@
+<content-header short="true">
```

Comment:
I'd suggest not making overrides for every product type unless the page is going to show and example of a custom layout.